.env.example

```diff
@@ -1,20 +1,23 @@
-NEXT_PUBLIC_SERVER_LINK=https://api.starknet.id
```

Comment:
Why are you editing this file everytime ? It's just a template, you shouldn't use it